src/react-components/room/PlacePopover.js

```diff
@@ -5,19 +5,20 @@ import { Popover } from "../popover/Popover";
 import { ToolbarButton } from "../input/ToolbarButton";
 import { ReactComponent as ObjectIcon } from "../icons/Object.svg";
 
-export function PlacePopoverButton({ items, onSelect }) {
+export function PlacePopoverButton({ items }) {
```

Comment:
I've mentioned this before but at some point I hope we find a clearer word for this menu than "Place". I suspect "Place" will give people the wrong impression of it being about "this place we're in" (aka the scene), not "place a pen in your hand or an object in the scene". Some alternatives to consider: 
- Add
- Create
- Load
- New
- Spawn
- Generate

Similarly, I think "Share" is overloaded in the mobile world and we might want to consider something like, "Broadcast" or "Stream"

Reply:
I'll bring this up with DPX. Thanks for flagging this :+1: